data/xml/schema.rnc

```diff
@@ -49,7 +49,7 @@ Paper = element paper {
    )
 }
 Meta = element meta {
-  (element booktitle { MarkupText }
+  (element booktitle { attribute short { text }?, MarkupText }
```

Comment:
Maybe add a comment that the short title should have no year because it is usually used together with the year field?